Wikipedia/Localizations/azb.lproj/Localizable.strings

```diff
@@ -45,35 +59,33 @@
 "search-titles" = "باشلیق‌لار";
 "search-recent-clear-cancel" = "وازگئچ";
 "search-recent-clear-delete-all" = "هامیسین سیل";
+"settings-title" = "تنظیملر";
+"settings-language" = "ویکی‌پدیا دیلی";
 "main-menu-title" = "آرتیق";
-"main-menu-language-title" = "ویکی‌پئدیادا $1 آختار";
```

Comment:
@niedzielski that's just github's html for some reason showing the dollar sign on the right - I just opened the file with Atom and it's properly on the left.

The failing test is actually the "html in the translation" test :)